src/shared/utils/flowSchemaNormalizer.ts

```diff
@@ -133,10 +135,30 @@ export const normalizeSchema = (
     data?.field
   )
 
-  const filteredTags = filterTags(dedupedTags, lowerCasedSearchTerm)
+  const dedupedMeasurements = dedupeArray(measurements)
+
+  dedupedMeasurements.sort((a, b) => a.localeCompare(b))
+
+  const filteredTags = filterTags(dedupedTags, lowerCasedSearchTerm).map(
```

Comment:
Yup, not sure why I thought this would be handled by the previous filter but thanks